src/SfResources.de.resx

```diff
@@ -1420,7 +1420,7 @@
     <value>Ist Vorlage</value>
   </data>
   <data name="Grid_SortDescription" xml:space="preserve">
-    <value>DRÜCKE ENTER, UM ZU CHATEN</value>
+    <value>Drücken Sie die Eingabetaste, um zu sortieren</value>
```

Comment:
Hi @prakashrajsf3945 ,

![image](https://github.com/user-attachments/assets/15ac6194-3faf-4a74-b11e-befd198ccfcc)

Check and change the casing like we did for English

Reply:
Yes.Modified.